# dock ID
1:
  position:
    x: 1.854648
    y: -1.5513322
  orientation:
    x: 0.0
    y: 0.0
    z: 0.016249007268174524
    w: 0.9998679761662531
  dock_bias:
    lateral: 0.0
    forward: -0.21

2:
  position:
    x: -0.9288063
    y: -0.6579091
  orientation:
    x: 0.0
    y: 0.0
    z: 0.006204306
    w: 0.999980753
  dock_bias:
    lateral: 0.0
    forward: -0.1
